File: SCC0600/exerciciosC/ex48.c
#include<stdio.h>

int main (void) {
    int size;
    scanf("%d", &size);
    int arr[size];
    for (int i = 0; i < size; i++) {
        scanf("%d", &arr[i]);
    }
    int maior = arr[0]; int indmaior = 0;
    int menor = arr[0]; int indmenor = 0;
    for (int i = 0; i < size; i++) {
        if (arr[i] > maior) {
            maior = arr[i];
            indmaior = i;
        }
        if (arr[i] < menor) {
            menor = arr[i];
            indmenor = i;
        }
    }
    arr[indmenor] = arr[0];
    arr[indmaior] = arr[size - 1];
    arr[0] = menor;
    arr[size - 1] = maior;
    for (int i = 0; i < size; i++) {
        printf("%d ", arr[i]);
    }
}

File: SCC0600/exercíciosPy/ex48.py
n = int(input())
vecinput = str(input())

def numerize (arr, rang):
  newarr = [0] * rang
  splited = arr.split(' ')
  for k in range(rang):
    newarr[k] = int(splited[k])
  return newarr

newvec = numerize(vecinput, n)

menor = newvec[0]
menorind = 0
for i in range(n):
  if newvec[i] < menor:
    menor = newvec[i]
    menorind = i
zeroelmenor = newvec[0]
newvec[0] = menor
newvec[menorind] = zeroelmenor

maior = newvec[0]
maiorind = 0
for i in range(n):
  if newvec[i] > maior:
    maior = newvec[i]
    maiorind = i
ultimoel = newvec[n - 1]
newvec[n - 1] = maior
newvec[maiorind] = ultimoel

print(*newvec)